README.md

```diff
@@ -8,6 +8,11 @@ On a Mac with [Homebrew](http://brew.sh/), run
 brew install cmake ninja boost libpng glfw3 libepoxy guile
 ```
 
+On Ubuntu:
+```
+sudo apt install cmake ninja-build libboost-all-dev libpng16-dev libglfw3-dev libepoxy-dev guile-2.0
```

Comment:
should `apt` be `apt-get`?  It *should* be `apt-get` on Debian.  Not sure what Ubuntu is like these days!